README.md

```diff
@@ -16,6 +16,23 @@ you might also like
 
 [Upcase]: https://upcase.com?utm_source=til
 
+## Contributing a TIL
```

Comment:
If people can't bother to read `CONTRIBUTING.md` then we can just close their PRs. Github even prompts you to check contribution guidelines when you open a PR 

![contributing](https://camo.githubusercontent.com/71995d6b0e620a9ef1ded00a04498241c69dd1bf/68747470733a2f2f6769746875622d696d616765732e73332e616d617a6f6e6177732e636f6d2f736b697463682f6973737565732d32303132303931332d3136323533392e6a7067)